nms/packages/magmalte/app/views/equipment/EquipmentGateway.js

```diff
@@ -92,32 +123,162 @@ const useStyles = makeStyles(theme => ({
   viewLabelText: {
     color: colors.primary.comet,
   },
+  listItemText: {
+    marginTop: '0px',
+  },
+  listItemPrimary: {
+    fontFamily: 'Inter',
+    fontSize: '16px',
+    color: colors.primary.mirage,
+    marginBottom: '18px',
+  },
+  listItemSecondary: {
+    fontFamily: 'Inter',
+    fontSize: '14px',
+    color: colors.primary.mirage,
+  },
+  listItem: {
+    alignItems: 'flex-start',
+  },
+  bulletList: {
+    padding: '8px',
+    listStyleType: 'disc',
+  },
 }));
 
 const UPGRADE_VIEW = 'UPGRADE';
+const EMPTY_STATE_OVERVIEW =
+  'The Access Gateway (AGW) provides network services and policy enforcement. In an LTE network,' +
+  ' the AGW implements an evolved packet core (EPC). It works with existing, unmodified commercial radio hardware.';
+const EMPTY_STATE_INSTRUCTIONS_STEP_2 =
+  'The Access Gateway (AGW) is configured and managed via the orchestrator and is part of a specific organization.' +
+  'This configuration is made through the NMS as part of adding a new gateway to the system. ';
 
 export default function Gateway() {
   const classes = useStyles();
+  const ctx = useContext(GatewayContext);
 
   return (
     <div className={classes.dashboardRoot}>
       <Grid container justify="space-between" spacing={3}>
-        <Grid item xs={12}>
-          <GatewayCheckinChart />
-        </Grid>
-        <Grid item xs={12}>
-          <Paper elevation={0}>
-            <EquipmentGatewayKPIs />
-          </Paper>
-        </Grid>
-        <Grid item xs={12}>
-          <GatewayTable />
-        </Grid>
+        {Object.keys(ctx.state).length > 0 ? (
+          <>
+            <Grid item xs={12}>
+              <GatewayCheckinChart />
+            </Grid>
+            <Grid item xs={12}>
+              <Paper elevation={0}>
+                <EquipmentGatewayKPIs />
+              </Paper>
+            </Grid>
+            <Grid item xs={12}>
+              <GatewayTable />
+            </Grid>
+          </>
+        ) : (
+          <>
+            <EmptyState
+              title={'Set up a Gateway'}
+              customIntructions={<GatewayEmptyStateList />}
+              instructions={''}
+              buttonText={'Add Gateway'}
+              // onClick={() => }
+              linkText={'Learn more about Access Gateway Configuration'}
+              link={
+                'https://docs.magmacore.org/docs/next/lte/deploy_config_agw'
+              }
```

Comment:
Why is this specified both here and in 238-240?

Reply:
It should not sorry!